File: atcoder/abc141A_weather_prediction.hs
-- uninhm
-- https://atcoder.jp/contests/abc141/tasks/abc141_a
-- implementation

import Data.List
import Data.Maybe

main = do
  let a = ["Sunny", "Cloudy", "Rainy"]
  li <- getLine
  putStrLn $ a !! (mod (1 + (fromJust $ elemIndex li a)) 3)

File: atcoder/abc141A_weather_prediction.ml
(*(* vicfred
 * https://atcoder.jp/contests/abc141/tasks/abc141_a
 * implementation
 * *)
Scanf.scanf "%s\n" (fun s ->
    if s = "Sunny" then "Cloudy"
    else if s = "Cloudy" then "Rainy"
    else "Sunny"
) |> Printf.printf "%s\n"

